packages/integrations/image/client.d.ts

```diff
@@ -0,0 +1,51 @@
+type InputFormat =
+	| 'heic'
+	| 'heif'
+	| 'avif'
+	| 'jpeg'
+	| 'jpg'
+	| 'png'
+	| 'tiff'
+	| 'webp'
+	| 'gif';
+
+interface ImageMetadata {
+	src: string;
+	width: number;
+	height: number;
+	format: InputFormat;
+}
+
+// images
+declare module '*.jpg' {
+	const metadata: ImageMetadata;
+	export default metadata;
+}
+declare module '*.jpeg' {
+	const metadata: ImageMetadata;
+	export default metadata;
+}
+declare module '*.png' {
+	const metadata: ImageMetadata;
+	export default metadata;
+}
+declare module '*.gif' {
+	const metadata: ImageMetadata;
+	export default metadata;
+}
+declare module '*.svg' {
+	const metadata: ImageMetadata;
+	export default metadata;
+}
+declare module '*.ico' {
+	const metadata: ImageMetadata;
+	export default metadata;
+}
```

Comment:
Should these two be here?

Reply:
Yep, great catch!  Looks like I missed going back to make sure the module defs match the `InputFormat` type